packages/manager/modules/mfa-enrollment/src/mfa-enrollment.routes.js

```diff
@@ -0,0 +1,15 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.mfaEnrollment', {
```

Comment:
Imo, it would be better for reuse than the state is delegated to the host app and not to the module